### PR TITLE
[148]  Add Page Navigation to your Pages- Fullstack

### PR DESCRIPTION
**Pull request by Mariamidowu**
Screenshot 2020-12-02 080833
Screenshot 2020-12-02 080915
Screenshot 2020-11-30 162639

### CODE REVIEW

**Review by Ireti203 — CHANGES_REQUESTED** [review]
Remove the debug.log in the html-forms folder

**Review by Ireti203 — COMMENTED** [review]
Why did you include a survey form in this task? The content is similar to the contact-form.

**Comment by Mariamidowu** [comment]
Gooday ma! I have deleted the survey form and also I was following the
video on the app, that's why it's there

**Comment by Ireti203** [comment]
Kindly resolve the conflicts on this branch.

**Comment by Ireti203** [comment]
Do you have challenges resolving the conflicts on this PR? @Mariamidowu

**Comment by Mariamidowu** [comment]
Yes ma. I have tried to remove it but it's not working.

**Comment by Mariamidowu** [comment]
Good day ma! I have removed the white space on the contact form and I have also deleted the files

File: submissions/idowuMariam/html-forms/product-page.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
```
Comment (adaugochi) [inline submissions/idowuMariam/html-forms/product-page.html:1]: Hello @Mariamidowu 

You don't have to create a product page in your `html-forms` folder. You already have this file in your `HTML` folder. Kindly link the pages on those already existing file. Like I said. You don't have to create a new file

File: submissions/idowuMariam/html-forms/Contact-form.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
```
Comment (adaugochi) [inline submissions/idowuMariam/html-forms/Contact-form.html:1]: Before working on `Add page Navigation` task, I believe that you must have work on creating a contact form page before. So, you don't have to create a new contact form page again. Kindly work on adding navigation links to those already existing pages

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Pull request closed without merging**